integration/testdata/kotlin/httpingress/Echokotlin.kt

```diff
@@ -0,0 +1,92 @@
+package ftl.echokotlin
+
+import ftl.builtin.HttpRequest
+import ftl.builtin.HttpResponse
+import kotlin.String
+import kotlin.Unit
+import xyz.block.ftl.Alias
+import xyz.block.ftl.Context
+import xyz.block.ftl.HttpIngress
+import xyz.block.ftl.Ignore
+import xyz.block.ftl.Method
+import xyz.block.ftl.Verb
+
+data class GetRequest(
+  val userID: String,
+  val postID: String,
+)
+
+data class GetResponse(
+  val message: String,
+)
+
+data class PostRequest(
+  @Alias("id") val userID: String,
+  val postID: String,
+)
+
+typealias PostResponse = Unit
+
+data class PutRequest(
+  val userID: String,
+  val postID: String,
+)
+
+typealias PutResponse = Unit
+
+data class DeleteRequest(
+  val userID: String,
+)
+
+typealias DeleteResponse = Unit
+
+class Echokotlin {
+  @Verb
+  @HttpIngress(
+    Method.GET,
+    "/echokotlin/users/{userID}/posts/{postID}",
+  )
+  fun `get`(context: Context, req: HttpRequest<GetRequest>): HttpResponse<GetResponse> {
+    return HttpResponse<GetResponse>(
+      status = 200,
+      headers = mapOf("Get" to arrayListOf("Header from FTL")),
+      body = GetResponse(message = "UserID, ${req.body.userID} : PostID ${req.body.postID}")
+    )
+  }
+
+  @Verb
+  @HttpIngress(
+    Method.POST,
+    "/echokotlin/users",
```

Comment:
Oh you're interpolating the runtime language into the module call in the tests?

Reply:
I think it would be preferable to have each language's implementation be identical though? So the tests can just be uniform?